src/main/java/core/basesyntax/WriteDataImpl.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+public class WriteDataImpl implements WriteData {
+
+    private String newPath;

```

Comment:
Consider passing `newPath` as a parameter to the `writeData` method instead of storing it as a class-level variable. This will allow more flexibility in writing to different files.